client/state/reader/streams/selectors/get-reader-stream.ts

```diff
@@ -1,13 +1,14 @@
 import 'calypso/state/reader/init';
+import { AppState } from 'calypso/types';
 
 const emptyStream = {
 	items: [],
 	pendingItems: { lastUpdated: null, items: [] },
 	lastPage: false,
-	isRequesting: false,
+	isRequesting: true, // `true` indicates that the stream is yet to fetch.
```

Comment:
Currently on new streams we have this as `false` which returns empty placeholder and re-renders the headers which causes jankiness in toggles so returning `true` as initial state fixes this issue.